src/app/models/workspace.server.model.ts

```diff
@@ -40,6 +41,16 @@ const compileWorkspaceModel = (db: Mongoose): IWorkspaceModel => {
     admin: 1,
   })
 
+  WorkspaceSchema.virtual('count').get(function (this: IWorkspaceSchema) {
```

Comment:
btw, cool use of virtuals! this is a good use case for it (although I'm not sure what it's used for? but I'll probably find out in later PRs)

Reply:
It's used in the FE, it's not necessary as clients can just use the `formIds.length`, but I felt this was just a nice-to-have

Reply:
I would say if it's not necessary then let's not add it in. creating derived properties makes it harder to keep everything in sync, and this doesn't give us any performance boost

Reply:
Okay then, I've updated the PR to remove the `count` virtual property!